### PR TITLE
Addition of search by name in OfficeFindFilter

### PR DESCRIPTION
**Pull request by EAndrey1091**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/1031
Added string NameIncludeSubstring prop to OfficeFindFilter, implemented filter by substring if the param not null or empty.

### CODE REVIEW

**Comment by BelehovEgor** [comment]
меня добавлять не надо

File: src/OfficeService.Data/OfficeRepository.cs [hunk 1]
```diff
@@ -62,13 +62,21 @@ public async Task<(List<DbOffice>, int totalCount)> FindAsync(OfficeFindFilter f
...
+        dbOffices = dbOffices.Where(x =>
+          x.Name.ToLower().Contains(filter.NameIncludeSubstring.ToLower()));
+      }
+
       if (filter.IsAscendingSort.HasValue)
       {
         dbOffices = filter.IsAscendingSort.Value 
         ? dbOffices.OrderBy(o => o.Name)
         : dbOffices.OrderByDescending(o => o.Name);
       }
```
Comment (a5x55555) [inline src/OfficeService.Data/OfficeRepository.cs]: Remove empty lines (78-79).

Comment (EAndrey1091) [reply src/OfficeService.Data/OfficeRepository.cs:77]: Done

Comment (EAndrey1091) [reply src/OfficeService.Data/OfficeRepository.cs]: Done

**Review by a5x55555 — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**